src/pages/CreateFormPage.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react';
+import { Formik } from 'formik';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { fetchFormCategories } from 'actions/formActions';
+import { getFormCategories } from 'selectors/formSelectors';
+import Select from 'components/formik/Select';
+import TextInput from 'components/formik/TextInput';
+import Button from 'components/common/Button';
+import Header from 'components/common/Header';
+import Title from 'components/common/Title';
+import Form from 'components/common/Form';
+
+function handleSubmit(values) {
+  alert(JSON.stringify(values));
+}
+
+const CreateFormPage = () => {
+  const formCategories = useSelector(getFormCategories);
```

Comment:
What about making these selectors custom hooks instead? Kinda like `useFormCategories`

Reply:
Can you explain a bit more why I should do this? I know that you use hooks to re-use code, but I don't really have code to share in the app here.
Maybe I'm missing something, thanks for the comment 👍